src/Rollbar.php

```diff
@@ -160,6 +160,16 @@ public static function fatalHandler()
             self::$logger->log(Level::CRITICAL, $exception, $extra);
         }
     }
+    
+    protected static function shouldLogFatal($last_error)
+    {
+        return
+            !is_null($last_error) &&
+            in_array($last_error['type'], self::$fatalErrors, true) &&
+            // don't log uncaught exceptions as they were handled by exceptionHandler()
+            isset($last_error['message']) &&
+            strpos($last_error['message'], 'Uncaught exception') !== 0;
```

Comment:
Oh I see what you are doing, the logic is a bit off for having two ands because you want to log the fatal if message is not set also